go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	github.com/containernetworking/cni v0.8.1
 	github.com/containernetworking/plugins v0.8.7
 	github.com/coreos/go-iptables v0.6.0
+	github.com/fsnotify/fsnotify v1.5.1
```

Comment:
nit: could we use the latest version (v1.5.4 at the moment)?

Reply:
Updated